### DISCUSSION TITLE
Example for Commando.js Framework [discord.js]

### DISCUSSION THREAD

**Discussion by SurgicalCrow** [helmasaur/horsengel-roulette#44] [Q&A] [answered]
I would be helpful if you tell me more about how to make it work in commando framework. I got the `!hr play` , I will just check if the user accepts or not in the play command file, but what is `!pan` ?

**Comment by helmasaur** [comment, marked as answer]
Ok, so I'll explain!

1. To start a game, you'd need to use the command `!hr [user]`.
2. To accept the duel, the user should type `!yes`
3. Each turn, to shoot, the user needs to write `!pan` ("pan" is the onomatopoeia of the noise of a gun shot in French).

To use it with Commando.js, you might need to create an empty file with an empty class for both `!yes` and `!pan` command with only the constructor filled.

I am working on bringing an update to the game to improve the user experience. It will normally be added the first half of May ☺️ !

I will also make a real documentation so it's easier to use!

If you have any questions or if something I wrote is not clear, do not hesitate to ask anything and I will try my best to answer!
